js/vnext/projection/cells.js

```diff
@@ -34,16 +34,8 @@ function translateRow(columnGroup, row, group) {
         const cell = row.item[col.name] || {};
         const classes = col.classes;
         const template = col[group + 'Template'];
-        let html = '';
-        if (cell.html) {
-          html = cell.html;
-        } else {
-          if (template) {
-            html = _.isFunction(template) ? template(group == 'head' ? columnGroup : row.item) : template;
-          } else {
-            html = _.isObject(cell) ? cell.name : cell;
-          }
-        }
+        const defaultHTML = _.isObject(cell) ? cell.name : cell;
+        const html = _.result(cell, 'html', template ? template(group == 'head' ? col : row.item) : defaultHTML);
```

Comment:
This can be easier, you don't need to care about the `cell.name`
```js
const html = _.result(cell, 'html', template(group === 'head' ? col : row.item));
```
is fine